go/ct/gen/accounts.go

```diff
@@ -32,17 +33,33 @@ func NewAccountGenerator() *AccountsGenerator {
 
 func (g *AccountsGenerator) Clone() *AccountsGenerator {
 	return &AccountsGenerator{
-		warmCold: slices.Clone(g.warmCold),
+		emptyAccounts: slices.Clone(g.emptyAccounts),
+		warmCold:      slices.Clone(g.warmCold),
 	}
 }
 
 func (g *AccountsGenerator) Restore(other *AccountsGenerator) {
 	if g == other {
 		return
 	}
+	g.emptyAccounts = slices.Clone(other.emptyAccounts)
 	g.warmCold = slices.Clone(other.warmCold)
 }
 
+func (g *AccountsGenerator) BindToAddressOfEmptyAccount(address Variable) {
+	c := emptyConstraint{address, true}
+	if !slices.Contains(g.emptyAccounts, c) {
+		g.emptyAccounts = append(g.emptyAccounts, c)
+	}
+}
+
+func (g *AccountsGenerator) BindToAddressOfNoneEmptyAccount(address Variable) {
+	c := emptyConstraint{address, false}
+	if !slices.Contains(g.emptyAccounts, c) {
+		g.emptyAccounts = append(g.emptyAccounts, c)
+	}
+}
+
 func (g *AccountsGenerator) BindWarm(address Variable) {
```

Comment:
To keep naming, BindToWarmAddress

Reply:
There is a long list of badly named functions here ... they should all be updated in one go. I think it is beyond the scope of this PR to fix those naming issues.